concent_api/core/message_handlers.py

```diff
@@ -1316,6 +1316,8 @@ def handle_send_subtask_results_verify(
         report_computed_task,
     )
 
+    validate_frames(compute_task_def['extra_data']['frames'])
```

Comment:
Shouldn't we validate this in other use cases too ? If yes, it should be in `validate_task_to_compute` function.

Reply:
I have asked @cameel https://github.com/golemfactory/concent/issues/537#issuecomment-402443056 and I'm waiting for answer :)

Reply:
@cameel already answered that we should check it always so I added to validate_task_to_compute